demos/graph-classification/README.md

```diff
@@ -0,0 +1,17 @@
+# Graph classification using StellarGraph
+
+[StellarGraph](https://github.com/stellargraph/stellargraph) allows for doing graph classification.
```

Comment:
possibly some alternate language to "allows for doing"? eg:

```suggestion
[StellarGraph](https://github.com/stellargraph/stellargraph) provides algorithms for graph classification.
```

Reply:
Yeah, that new phrasing is way better...